src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,170 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    public class Node<K, V> {
+        private final int hash;
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    private final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private final double DEFAULT_LOAD_FACTOR = 0.75;
+    private int threshold;
+    private boolean isResized = false;
```

Comment:
The `isResized` variable is declared but never used in the code. If it's not needed, consider removing it to clean up the code.